[2022-03-14 12:05:20,266] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-13T12:04:19.461189+00:00 [queued]>
[2022-03-14 12:05:20,283] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-13T12:04:19.461189+00:00 [queued]>
[2022-03-14 12:05:20,285] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 12:05:20,286] {taskinstance.py:1239} INFO - Starting attempt 1 of 2
[2022-03-14 12:05:20,287] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-14 12:05:20,311] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): cat.cat_get_data> on 2022-03-13 12:04:19.461189+00:00
[2022-03-14 12:05:20,319] {standard_task_runner.py:52} INFO - Started process 876 to run task
[2022-03-14 12:05:20,326] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'tutorial', 'cat.cat_get_data', 'scheduled__2022-03-13T12:04:19.461189+00:00', '--job-id', '816', '--raw', '--subdir', 'DAGS_FOLDER/test_dag.py', '--cfg-path', '/tmp/tmpzb0hv2o8', '--error-file', '/tmp/tmptjzxcgcs']
[2022-03-14 12:05:20,328] {standard_task_runner.py:77} INFO - Job 816: Subtask cat.cat_get_data
[2022-03-14 12:05:20,454] {logging_mixin.py:109} INFO - Running <TaskInstance: tutorial.cat.cat_get_data scheduled__2022-03-13T12:04:19.461189+00:00 [running]> on host ff48034dfdbf
[2022-03-14 12:05:20,557] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=***@example.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=tutorial
AIRFLOW_CTX_TASK_ID=cat.cat_get_data
AIRFLOW_CTX_EXECUTION_DATE=2022-03-13T12:04:19.461189+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-13T12:04:19.461189+00:00
[2022-03-14 12:06:20,565] {python.py:175} INFO - Done. Returned value was: None
[2022-03-14 12:06:20,581] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=tutorial, task_id=cat.cat_get_data, execution_date=20220313T120419, start_date=20220314T120520, end_date=20220314T120620
[2022-03-14 12:06:20,636] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-14 12:06:20,691] {local_task_job.py:264} INFO - 1 downstream tasks scheduled from follow-on schedule check
